mitodl/mitxpro
# Save order on enrollment objects

#### Pre-Flight checklist

#### What are the relevant tickets?
Closes #289

#### What's this PR do?
Adds a foreign key to `Order` for `CourseEnrollment` and `ProgramEnrollment`

#### How should this be manually tested?
- After the migration script is run, existing enrollments should have a populated `order` attribute.
- Purchase a new course run and a new program.  The generated enrollments should all have a populated `order` attribute pointing to your orders.
